database/seeds/BeatmapsetTableSeeder.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use DB;
+use Illuminate\Database\Seeder;
+use Illuminate\Database\Eloquent\Model;
+
+class BeatmapsetTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+		Model::unguard();
+		$faker = Faker\Factory::create();
+		
+		for ($i = 0; $i < 5; $i++) {
+			
+			$faker->seed($i);
```

Comment:
Putting the seed inside the loop was intended so that everyone would generate the same sample data. It also allows for me to have the same user_id in multiple tables.